modules/engines/engine-xsjs/src/main/java/com/sap/xsk/engine/XSKJavascriptEngineRestService.java

```diff
@@ -94,8 +95,8 @@ public Response executeRhinoServicePost(@PathParam("path") String path) {
   @Path("/{path:.*}")
   @ApiOperation("Execute Server Side JavaScript HANA XS Classic Resource")
   @ApiResponses({@ApiResponse(code = 200, message = "Execution Result")})
-  public Response executeRhinoServicePut(@PathParam("path") String path) {
-    return executeRhinoServiceGet(path);
+  public Response executeServicePut(@PathParam("path") String path) {
+    return executeServiceGet(path);
```

Comment:
Move the implementation of the `executeServiceGet` method into a new private method and use it in all routes